### DISCUSSION TITLE
I dont have an email address And I cant get  the pack

### DISCUSSION THREAD

**Discussion by mkrtik** [community/community#59717] [GitHub Education]
### Hi everyone,

Hey guys! So I had applied for the pack in February of this year. And I was approved But for some reason it got expired after around four months today. I reapplied for the pack. My school hasnt given an email. So I used my Photo id card which includes my schools name , my name and date as well. But It got rejected saying that it requires an email or it needs permission to access my device's camera. How do I give it Camera access ? It doesnt have an option for that? 
Please help.

**Comment by samus-aran** [comment]
Hi @kartik01m 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.
As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. 

In this case it would be the Academic email(s) which has been successfully associated with this school.

We would recommend to you speak to your institution regarding to acquiring an email address to proceed.
